src/drivers/mct/shr/seq_comm_mct.F90

```diff
@@ -1129,14 +1124,15 @@ end subroutine seq_comm_setnthreads
   integer function seq_comm_getnthreads()
 
     implicit none
+#ifdef _OPENMP
     integer :: omp_get_num_threads
-    character(*),parameter :: subName =   '(seq_comm_getnthreads) '
 
     seq_comm_getnthreads = -1
-#ifdef _OPENMP
 !$OMP PARALLEL
     seq_comm_getnthreads = omp_get_num_threads()
 !$OMP END PARALLEL
+#else
+    seq_comm_getnthreads = -1
```

Comment:
Redundant with the initialization on line 1130 - you can remove one or the other

Reply:
This is not redundant as the compiler will only see one of them.

Reply:
Oh, sorry, I misread this code. Really what I should have said is that line 1130 is unnecessary because it will get set on line 1132. But this isn't a big deal.